docs/extensions/capabilities/common-capabilities.md

```diff
@@ -0,0 +1,258 @@
+# Common Capabilities
+
+Common Capabilities are important building blocks for your extensions. Almost all extensions use some of these functionalities. Here is how you can take advantage of them.
+
+## Main Extension
```

Comment:
I feel there's going to be some confusion about what we call an extension. An extension (as in a package we get from the extension marketplace) consists of one or both of the extension classes/objects "Main Extension" and "Renderer Extension". What about "Main Extension Object", or "Main Extender", or something else to provide distinction?

Reply:
I think we should clarify these in extension anatomy? @nevalla wdyt?

Reply:
Yes, there are room for improvements there.